docs/csharp/misc/cs0225.md

```diff
@@ -8,7 +8,7 @@ helpviewer_keywords:
 ms.assetid: 0b0cd72b-c47a-44d1-9b27-d1a1fad06807
 ---
 # Compiler Error CS0225
-The params parameter must be a single dimensional array  
+The params parameter must be a single-dimensional array  
```

Comment:
nit: strictly speaking, it's a quote of the compiler message:
https://github.com/dotnet/roslyn/blob/d834ca2ee9efc47c57246163f6649227bb0f718f/src/Compilers/CSharp/Portable/CSharpResources.resx#L967

So, the compiler message itself misses hyphen. 
@rpetrusha @mairaw related question: how far can we go in correcting the compiler messages? This particular case with hyphen looks fine.

Reply:
Ha, good catch @pkulikov. I’d probably keep product strings as-is and change if we change in the product.

Reply:
Thanks for catching this, @pkulikov. I agree, @mairaw; we should keep the string as-is. I've closed this PR.